### ISSUE TITLE
Update various components in the stack and new Release

### ISSUE THREAD

**Issue by yatharthranjan** [RADAR-base/RADAR-Docker#121]
New versions have been released for major components like radar-backend, mongodb-connector, hdfs-connector, etc.
Also include the latest version of hdfs-restructure jar
Also make a release of RADAR platform 2.0.0 after these are updated and new confluent version is used -- refer #120

**Comment by blootsvoets** [comment]
Perhaps a version 1.1.0 is in order first? It's just a version bump with some additional scripts since 1.0.0 right?

**Comment by yatharthranjan** [comment]
Yes i think so. Nothing major since then. 1.1.0 should be fine. Thanks
